.env.testnet.local

```diff
@@ -44,7 +44,6 @@ THE_GRAPH_API_KEY=106677 # This key is for developing purposes only and have lim
 
 DAO_GRAPH_URL=https://api.studio.thegraph.com/query
 DAO_GRAPH_ID=rootstock-collective-governance-subgraph/version/latest
-DAO_GRAPH_API_KEY=56839  # This key is for developing purposes only and have limited amount of queries
```

Comment:
We should not remove this, this is for local development.

Reply:
But I'm only setting one env variable in AWS and that is `THE_GRAPH_API_KEY`. Having both is somewhat confusing, the application should be configured to expect one for each envs.

Reply:
@david-iov each graph implementation is different. TOK has their own implementation. DAO has their own implementation. If you want to unify both envs, which will require more effort, then we will have to setup some sort of plan to do this; and I don't think the scope of this PR is for this.

We can create a ticket to follow up on this; for now let's keep it there.